lib/unitary_calculator_avx.h

```diff
@@ -17,21 +17,22 @@
 
 #include <immintrin.h>
 
-#include <algorithm>
 #include <complex>
 #include <cstdint>
+#include <functional>
+#include <vector>
 
-#include "bits.h"
+#include "simulator.h"
 #include "unitaryspace_avx.h"
 
 namespace qsim {
 namespace unitary {
 
 /**
- * Quantum circuit unitary calculator with AVX vectorization.
+ * Quantum circuit simulator with AVX vectorization.
```

Comment:
Nit: word change here and other unitary_calulcator_*** variants.

Reply:
Fixed.